src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,62 @@
+.nav {
+  &__list {
+    display: flex;
+
```

Comment:
delete redundant empty lines

Reply:
@serentinos Oh, I see, it's just for better readability
In this way you can leave it as it is, don't need to do any changes ⭐️